packages/vue-cli-plugin-vuetify/generator/templates/v3/src/plugins/vuetify.js

```diff
@@ -0,0 +1,18 @@
+import "@mdi/font/css/materialdesignicons.css";
+import { createVuetify } from "vuetify";
+import { aliases, mdi } from "vuetify/src/iconsets/mdi";
+import { fa } from "vuetify/src/iconsets/fa-svg";
+
+export default createVuetify({
+  // lang: {
```

Comment:
remove commented out code

Reply:
Do we need any of the commented out portion at the moment?